src/main/java/it/gov/pagopa/pu/pagopapayments/enums/RegistryEventCategory.java

```diff
@@ -0,0 +1,5 @@
+package it.gov.pagopa.pu.pagopapayments.enums;
+
+public enum RegistryEventCategory {
+  INTERNAL, INTERFACE
```

Comment:
let's use the exact values reported on doc:
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1801519115/Giornale+degli+eventi#2.-Giornale-degli-eventi-pagoPA

```suggestion
  INTERNO, INTERFACCIA
```